scripts/travis/vtk.sh

```diff
@@ -2,7 +2,7 @@
 set -e
 if [ ! -d "$HOME/VTK-Install/include" ]; then
 	cd $HOME
-	wget http://www.opengeosys.org/images/dev/vtk-6.1.0.tar.gz
+	wget --no-check-certificate http://www.opengeosys.org/images/dev/vtk-6.1.0.tar.gz
```

Comment:
In the file "ThirdPartyLibVersions.cmake" vtk version points to 6.3. Here the version 6.1.0 is downloaded. Is this intended?